client/src/Component/NavbarComponent.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Navbar, Nav, Container } from "react-bootstrap";
+
+
+const NavbarComponent = () => {
+	return (
+		<Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
```

Comment:
```suggestion
		<Navbar collapseOnSelect expand="sm" bg="dark" variant="dark">
```
With `expand="lg"`, we get the hamburger menu button at medium sized devices even though they can comfortably show the `Home` and `Calendar` links on the navbar. We could update this to `sm` instead so that the navbar links change to a hamburger menu at a smaller screen size